aldryn_jobs/cms_wizards.py

```diff
@@ -0,0 +1,162 @@
+        return False
+
+
+class JobCategoryWizard(ConfigCheckMixin, Wizard):
+    perm_string = "aldryn_jobs.add_jobcategory"
+
+
+class JobOpeningWizard(ConfigCheckMixin, Wizard):
+    perm_string = "aldryn_jobs.add_jobapplication"
+
+    def user_has_add_permission(self, user, **kwargs):
+        """
+        Return True if the current user has permission to add a JobOpening and
+        there is at least one JobCategory.
+        :param user: The current user
+        :param kwargs: Ignored here
+        :return: True if user has add permission, else False
+        """
+        base_perm = super(JobOpeningWizard, self).user_has_add_permission(
+            user, **kwargs)
+        return base_perm and JobCategory.objects.exists()
+
+
+class CreateJobCategoryForm(BaseFormMixin, TranslatableModelForm):
+    """
+    The ModelForm for the JobCategory wizard. Note that JobCategory has few
+    translated fields that we need to access, so, we use TranslatableModelForm
+    """
+
+    class Meta:
+        model = JobCategory
+        fields = ['name', 'slug', 'app_config']
+
+    def __init__(self, **kwargs):
+        super(CreateJobCategoryForm, self).__init__(**kwargs)
+        # If there's only 1 app_config, don't bother show the
+        # app_config choice field, we'll choose the option for the user.
+        app_configs = JobsConfig.objects.all()
+        # check if app config is apphooked
+        app_configs = [app_config
+                       for app_config in app_configs
+                       if namespace_is_apphooked(app_config.namespace)]
+        if len(app_configs) == 1:
+            self.fields['app_config'].widget = forms.HiddenInput()
+            self.fields['app_config'].initial = app_configs[0].pk
+
+
+class CreateJobOpeningForm(BaseFormMixin, TranslatableModelForm):
+    """
+    The ModelForm for the Job Opening wizard. Note that JobOpening has few
+    translated fields that we need to access, so, we use TranslatableModelForm
+    """
+
+    content = forms.CharField(
+        label="Content", help_text=_("Optional. If provided, will be added to "
+                                    "the main body of the Opening content."),
+        required=False, widget=forms.Textarea())
+
+    class Meta:
+        model = JobOpening
+        fields = ['title', 'category', 'is_active', 'lead_in',
+                  'publication_start', 'publication_end',
+                  'content', 'can_apply', ]
+
+    def __init__(self, **kwargs):
+        super(CreateJobOpeningForm, self).__init__(**kwargs)
+
+        # If there's only 1 category, don't bother show the empty label (choice)
+        if JobCategory.objects.count() == 1:
+            self.fields['category'].empty_label = None
+        self.fields['publication_start'].help_text = _(
+            'Date Acceptable Formats: 2015-11-11, 11/11/2015, 11/11/15')
+        self.fields['publication_end'].help_text = _(
+            'Date Acceptable Formats: 2015-11-11, 11/11/2015, 11/11/15')
```

Comment:
These are very poor example dates to use. You should use dates where it is obvious which segment is the month and which is the day, like 2015/12/31.